openedx/features/journals/api.py

```diff
@@ -1,22 +1,23 @@
 """
 APIs providing support for Journals functionality.
 """
-from urlparse import urljoin, urlsplit, urlunsplit
-import logging
+from __future__ import absolute_import
 import hashlib
+import logging
 import six
 
+from six.moves.urllib.parse import urljoin, urlsplit, urlunsplit
```

Comment:
jenkins pylint is failing on this line, saying it can't import it. However, jenkins python is quite happy. Is pylint using a different version of six? @jmbowman